index.js

```diff
@@ -108,6 +108,17 @@ function outputChangelog (argv, cb) {
   })
 }
 
+function handleExecError (err, stderr) {
```

Comment:
I thought we should handle `err` and `stderr` differently?
the problem is with `stderr` shouldn't terminate the program. But for `err` we keep the same behaviour. No?